### ISSUE TITLE
crashed whilist exploring

### ISSUE THREAD

**Issue by redalpha00** [MehVahdJukaar/TrickOrTreatMod#30]
[crash-2023-04-25_09.54.34-server.txt](https://github.com/MehVahdJukaar/TrickOrTreatMod/files/11323864/crash-2023-04-25_09.54.34-server.txt)

**Comment by MehVahdJukaar** [comment]
Optifine

**Comment by redalpha00** [comment]
thank you, man I can't run opitfine with anything no more

**Comment by MehVahdJukaar** [comment]
They've had this issue for more than a year. It breaks a forge class. It's incompatible with forge itself. Optofine shouldn't be used over th years it had a reputation for breaking shit all over the place because of th way it's msde

**Comment by redalpha00** [comment]
yeah i'm currently looking for alternatives that aren't a head ache to install but so far no luck

**Comment by MehVahdJukaar** [comment]
Ther s a fix on CF for that particular bug

**Comment by redalpha00** [comment]
prey tell

**Comment by MehVahdJukaar** [comment]
Sodium and it's ports. Also that specific bug has a mod fix on cf
